### PR TITLE
Preceding and succeeding titles, Streamline SRS

### LINKED ISSUE #142: Handle preceding/succeeding titles
https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/p/instance-preceding-succeeding-titles.html 

Example of SUL record: https://folio-test.stanford.edu/inventory/view/57e5625f-3747-529b-af20-4d07cfc647 

Expected behavior: Preceding or succeeding title should appear in the Instance> Title Data accordion with a link to the relevant related Instance. 

Example of Lane record with expected behavior: https://folio-test.stanford.edu/inventory/view/0455c665-c9c1-5313-aa41-a7efa186a8

TTD:
- [x] Fork https://github.com/FOLIO-FSE/folio_migration_tools and refactor `create_preceding_succeeding_titles` method ~~to remove conditions that require ISSN/ISBN~~ to output entities to file for subsequent processing.
- [x] Use forked migration_tools repo when deploying/running libsys-airflow (this will allow the population of the inventory record without creating the links between records).
- [x] ~~Find out of stakeholders will require~~ Parity will not require linkages on day 1. Create a new issue to be worked on later to create record linkages. 
- [x] ~~Propose upstream change to FOLIO-FSE?~~ FOLIO-FSE will be coming up with some changes that may help us with this, so BOLO...

### PR DESCRIPTION
**Pull request by jgreben**
Partially fixes https://github.com/sul-dlss/FOLIO-Project-Stanford/issues/142:
- Save preceding_succeeding_titles output to files for later processing and loading
- Also adds the streamlining of SRS record loading that was in jeremnelson personal branch.

### CODE REVIEW

**Comment by jgreben** [comment]
I wasn't quite sure what to test here for the pre/succeeding titles. There are not any pre-existing tests for the conversion/mapping of the 780's but I also did not modify any of the RulesMapper code, so I just did a basic test that the pre/succeeding titles entity gets created and logged. There were also no pre-existing tests for the FolderStructure and writing of files, so I skipped that aspect as well. ¯\_(ツ)_/¯

File: src/folio_migration_tools/folder_structure.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from pathlib import Path
 
 from folio_uuid.folio_namespaces import FOLIONamespaces
+from folio_migration_tools.library_configuration import LibraryConfiguration
```
Comment (jermnelson) [inline src/folio_migration_tools/folder_structure.py]: Is this import needed?

File: src/folio_migration_tools/marc_rules_transformation/rules_mapper_base.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import time
 import uuid
 from textwrap import wrap
+from folio_migration_tools.folder_structure import FolderStructure
```
Comment (jermnelson) [inline src/folio_migration_tools/marc_rules_transformation/rules_mapper_base.py:7]: I think this is a duplicate import statement (see line 21) and can be removed. The second import is located within the context of the other imports.

**Review by jermnelson — CHANGES_REQUESTED** [review]

File: src/folio_migration_tools/marc_rules_transformation/rules_mapper_base.py (hunk 2 above)
Comment (jgreben) [reply src/folio_migration_tools/marc_rules_transformation/rules_mapper_base.py:7]: Fixed lints

**Review by jermnelson — APPROVED** [review]

**Pull request merged**